Testscripts/Linux/dpdkPmdsBasicCheck.sh

```diff
@@ -147,8 +147,16 @@ SetTestStateRunning
 mkdir -p $LOGDIR
 IFS=',' read -r -a pmd_array <<< "$pmds"
 exitcode=0
-eth1_vfname=$(ip addr|grep 'master eth1'|awk -F ': ' '{print $2}')
+eth1_vfinfo=$(ip addr show master eth1)
```

Comment:
some images' vf ethernet name is not eth1.

Reply:
This is an excellent point, when I rewrite this test I will not assume the interface name. This patch is just updating the code that was there to fail more gracefully, the legacy code does assume eth1 for some reason.